packages/react-native/src/Authenticator/hooks/useFieldValues/useFieldValues.ts

```diff
@@ -25,7 +25,7 @@ export default function useFieldValues<FieldType extends TypedField>({
   const sanitizedFields = useMemo(() => {
     if (!Array.isArray(fields)) {
       logger.warn(
-        `Invalid fields type of ${typeof fields} passed to ${componentName}. fields must be of type array.`
+        `Invalid fields type of ${typeof fields} passed to ${componentName}. Fields must be of type array.`
```

Comment:
The variable name is `fields`, the lowercase is intentional 😅

Reply:
Maybe 
```suggestion
        `Invalid fields type of ${typeof fields} passed to ${componentName}. \`Fields\` must be of type array.`
```

Reply:
(edited for typo)

Reply:
Since this change is not relevant to the PR, let's remove it. If we want to change the messages in this file they should all be done and discussed in its own PR